docs/streams/tutorial.html

```diff
@@ -62,8 +62,8 @@
     </pre>
 
     <p>
-        The <code>pom.xml</code> file included in the project already has the Streams dependency defined,
-        and there are already several example programs written with Streams library under <code>src/main/java</code>.
+        <strong>Important:</strong> You must manually update the setting of <code>&lt;kafka.version&gt;</code> in the generated <code>pom.xml</code> file from <code>0.11.0.1-SNAPSHOT</code> to <code>0.11.0.1</code>.
```

Comment:
Maybe worth mentioning that this has been improved/fixed for the next release? That way people won't think we made it hard on purpose. :)